Changelog.md

```diff
@@ -4,6 +4,8 @@ __New Features__
 - Allows multiple versions of a given program (e.g., ENERGY STAR 3.2 and 3.3) to be calculated in a single call.
   - **Breaking change**: Output directories and files have been reorganized/renamed (output file contents are not changed in any way). 
 - Allows specifying the number of parallel processors to use for simulations with `-n <NUM>` or `--num-proc <NUM>`.
+- Updates to ENERGY STAR MFNC Rev 05.
+  - **Breaking change**: Update U-factor for floors over 'other multifamily buffer space'.
```

Comment:
This is not a breaking change, a breaking change means that a user of the software has to do something different in how they create the HPXML files or call OS-ERI. And I'd say this is a bugfix, not a new feature, since we already said we implemented Rev 05 in v1.9.4. How about:

```suggestion

__Bugfixes__
- Fixes U-factor for floors over 'other multifamily buffer space' per ENERGY STAR MFNC Rev 05.
```